### DISCUSSION TITLE
Cloud-based Serverless deployment?

### DISCUSSION THREAD

**Discussion by Swampcritter** [YOURLS/YOURLS#2897] [Q&A]
Is it possible that YOURLS be installed on AWS serverless platform? I was reading about an application called Bref (bref.sh) as it provides PHP capabilities to serverless platforms. The YOURLS database could then utilize AWS RDS offerings like MySQL or MariaDB.

Didn't know if anyone has attempted trying to deploy YOURLS in this way.

**Comment by ozh** [comment]
It's certainly possible -- it's just PHP and MySQL. An easy option might be to use the Docker version of YOURLS ?
